Makefile

```diff
@@ -49,7 +49,7 @@ watch-test: node_modules
 .PHONY: watch-tests
 watch-tests:
 	nodemon -w gatsby-node.js \
-	--exec "$(MAKE) -C tests/gatsby-starter-default clean watch"
+	--exec "$(MAKE) -C tests/gatsby-starter-default watch"
```

Comment:
@ajayns "clean" is no longer required, because of incremental build support; `make clean` could be called separately to start from scratch

Reply:
Got it, we're not clearing cache each time so as to support incremental build yes?